src/tests/e2e/login.post.test.ts

```diff
@@ -0,0 +1,61 @@
+import 'dotenv/config';
+import request from 'supertest';
+import app from '../../app';
+import { mockPassword, mockUsername, mockBasicAuthString } from '../mockObjects/mockUser';
+import { setupE2E } from '../setupTests';
+import { ReasonPhrases, StatusCodes } from 'http-status-codes';
+import { userService } from '../../services/userService';
+import { mockComparePassword, mockGenerateTokens, mockGetUser } from '../mockObjects/mockUserService';
+
+setupE2E('login-e2e', () => {
```

Comment:
I don't see validation in login endpoint, but it is defined in userValidator
<img width="381" alt="image" src="https://user-images.githubusercontent.com/59976377/183393211-c22c498a-9fda-4335-baf8-dfa9df6bf71a.png">

<img width="660" alt="image" src="https://user-images.githubusercontent.com/59976377/183393016-05ed5b9b-ed8d-41fd-8a7e-71afdf878b16.png">

Reply:
Add validation to login and add tests for validating username and password